app/views/endemics/numbers.njk

```diff
@@ -8,7 +8,6 @@
     href: backLink,
 	  attributes: {id: "back"}
   }) }}
-  {{ nameSbiTag({name: organisation.name, sbi: organisation.sbi}) }}
```

Comment:
I think you can delete the nameSbiTag file and remove it from the layout file also. It was only used in these 3 places.